script/release/uploaders/upload.py

```diff
@@ -205,15 +213,17 @@ def purify_extra_data(mm, offset, length):
 
       if header_id in (EXTENDED_TIME_DATA, UNIX_EXTRA_DATA):
         values[0] = STRIPZIP_OPTION_HEADER
-        for i in xrange(2, len(values)):
+        for i in range(2, len(values)):
           values[i] = 0xff
         extra_struct.pack_into(mm, offset, *values)
-      elif header_id != STRIPZIP_OPTION_HEADER:
-        return False
+      if header_id == ZIP64_EXTRA_HEADER:
+        assert header_length == 16
+        values = list(zip64_extra_struct.unpack_from(mm, offset))
+        header_id, header_length, uncompressed_size, compressed_size = values[0], values[1], values[2], values[3]
```

Comment:
```suggestion
        header_id, header_length, uncompressed_size, compressed_size = values
```

Reply:
Isn't values unpacking without the index only working on python3 (it is not python2 compatible)

Reply:
```
$ python2.7
>>> a, b, c = [1,2,3]
>>> a
1
>>> b
2
>>> c
3
```

Reply:
Oh right, I confused that with the `a, b, *rest = [1,2,3,4]` syntax

```
>>> a, b, *rest = [1,2,3,4]
  File "<stdin>", line 1
    a, b, *rest = [1,2,3,4]
          ^
SyntaxError: invalid syntax
```